climakitae/explore/vulnerability.py

```diff
@@ -1181,16 +1187,22 @@ def cava_data(
         case "calculate" | "both":
             if separate_files:
                 for _, data_point in enumerate(metric_data):
-                    lat_str = str(round(data_point.lat.item(), 3)).replace(".", "")
-                    lon_str = str(round(data_point.lon.item(), 3)).replace(".", "")
+                    lat_str = str(round(data_point.lat.item(), 6)).replace(".", "")
+                    lon_str = str(round(data_point.lon.item(), 6)).replace(".", "")
                     _export_no_e(
                         data_point,
-                        filename=f"{clean_params['calc_name']}_{lat_str}N_{lon_str}W",
+                        filename=(
+                            f"{clean_params['calc_name']}_{lat_str}N_{lon_str}W"
+                            if file_name is None
```

Comment:
Longitude in the US is negative. So this will be “_-117.161087W” which is fine, but just wanted to mention since it's a bit odd with the underscore followed by a hyphen. I don’t have a better suggestion, just pointing this out.